motion/p5js/individual/index.md

```diff
@@ -0,0 +1,11 @@
+---  
+layout: p5js
+author: Ludwig Frank
+author-url: ludwigfrank.com
+title: individual
+tags: P5.js, Motion, Law of Similarity
+permalink: /motion/individual/
+summary: How motion leads to perceived similarity.
+---  
+
+![Preview](./out.png)  
```

Comment:
This is actually not needed. The code from the index.js will be added to the site. If you assign the canvas the sketch ID like this:  

```
  var canvas = createCanvas(100, 100);
  canvas.parent('sketch'); 
```